gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
+            general.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+      </NxDescriptionList>
       <NxP>
-        Documentation of all React components available to consumers of RSC, including demonstrations, example
-        code, and full documentation of relevant React <NxCode>props</NxCode>. Examples here
-        demonstrate basic usage of each component along with some modicum of supporting code - for example,
-        code tracking the toggle state of a <NxCode>NxCheckbox</NxCode>. This supporting code
-        should be taken as example only. Depending on the architectural needs of the consuming application, there
-        are many different ways that state management and other supporting concerns could be handled.
+        In addition to the groupings within the sidebar, you will notice that each page entry has an icon which
+        indicates the type of item documented on that page:
       </NxP>
-      <NxH4>Guidelines</NxH4>
-      <NxP>Miscellaneous advice for RSC consumers</NxP>
-      <NxH4>Styles - HTML Elements</NxH4>
-      <NxP>
-        Demonstrations of raw CSS classes that are available to consumers of RSC. Some of these classes are also
-        available as React components, but are documented here as well for lower-level/non-React use.
-      </NxP>
-      <NxH4>Styles - Mixins &amp; Helpers</NxH4>
-      <NxP>
-        Documentation of SCSS mixins and CSS helper classes available to consumers of RSC.
-      </NxP>
-      <NxH4>Layout Examples</NxH4>
-      <NxP>
-        Explanations of broader layout concepts that includes groups of components and/or styles working together.
-      </NxP>
-      <NxH4>JavaScript &amp; TypeScript Utilities</NxH4>
-      <NxP>Utility function and datatypes available to users of RSC.</NxP>
+      <NxDescriptionList>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faReact} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Documentation for a React component or family of closely related React components. Typically includes
+            technical details about the props that may be passed to the component along with live examples of its use.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faCode} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Documentation for usage of bare CSS classes within HTML. Some RSC "components" consist almost entirely of
+            styling with little or no special behavior that must be implemented. For these cases, the bare classes
+            are themselves a documented and stable part of the RSC API, allowing them to be used even in applications
+            which do not use React. Many of these components are also exposed via React "convenience components"
```

Comment:
Suggesting reword for this sentence:  For these cases, the bare classes are themselves a documented and stable part of the RSC API, allowing them to be used even in applications which do not use React.